addons/event/views/event_registration_views.xml

```diff
@@ -259,17 +269,38 @@
         </field>
     </record>
 
+    <record id="event_registration_view_attendee_search" model="ir.ui.view">
```

Comment:
Can't we use event_registration_view_search_event_specific ? Seems we are getting too much different views vaguely specific ... I would prefer having one search view that is generic, one that is event-specific, and normally that's all.

Moverover: no exception to ask and migration issues.